frontend/src/container/Tracim.jsx

```diff
@@ -752,6 +753,10 @@ export class Tracim extends React.Component {
           />
         )}
 
+        {((!this.props.user.username && Cookies.get(COOKIE_FRONTEND.SHOW_USERNAME_POPUP) === 'true' && this.props.user.config.display_username_popup !== 'false') &&
+          <CardPopupUsername />
+        )}
+
```

Comment:
```suggestion
        {((!props.user.username && Cookies.get(COOKIE_FRONTEND.SHOW_USERNAME_POPUP) === 'true' && props.user.config.display_username_popup !== 'false') &&
          <CardPopupUsername />
        )}

```



The hard coded strings 'true' and 'false' can't be consts or real booleans ?